donor-portal/src/pages/home/index.tsx

```diff
@@ -25,45 +25,86 @@ export default function Home() {
+            src="assets/images/slredcross_logo.png"
             alt="donating medical supplies"
           />
         </div>
         <div className="home-description-text">
           <h3>An Acute Medical Emergency</h3>
           <p>
-            <strong>
-              Sri Lanka lacks a coordinated response to address the overall
-              medical needs of hospitals. Given the limited resources it is
-              essential to optimize the demand and distribution of supplies to
-              help avoid possible duplication, address gaps and avoid
-              detrimental outcomes to citizens most in need.
-            </strong>
+            Sri Lanka lacks a coordinated response to address the overall
+            medical needs of hospitals. Given the limited resources it is
+            essential to optimize the demand and distribution of supplies to
+            help avoid possible duplication, address gaps and avoid detrimental
+            outcomes to citizens most in need.
           </p>
           <h3>An Alternative Approach</h3>
           <p>
-            <strong>
-              ELIXIR is a platform designed to meet this need. It is operated by
-              the Sri Lanka Red Cross along with a network of volunteers and
-              professionals to help improve the efficacy of donations. In
-              partnership with the Sri Lanka Medical Supplies Division we have
-              compiled aid packages that aggregate needs across multiple
-              hospitals to help achieve bulk price discounts from suppliers. Our
-              volunteers help negotiate prices and coordinate invoicing and
-              shipping logistics. ELIXIR facilitates transparency and
-              accountability throughout the entire supply chain.
-            </strong>
+            ELIXIR is a platform designed to meet this need. It is operated by
+            the Sri Lanka Red Cross along with a network of volunteers and
+            professionals to help improve the efficacy of donations. In
+            partnership with the Sri Lanka Medical Supplies Division we have
+            compiled aid packages that aggregate needs across multiple hospitals
+            to help achieve bulk price discounts from suppliers. Our volunteers
+            help negotiate prices and coordinate invoicing and shipping
+            logistics. ELIXIR facilitates transparency and accountability
+            throughout the entire supply chain.
           </p>
           <h3>Please Donate</h3>
           <p>
-            <strong>
-              Together we can do more. We are accepting donations from both
-              organizations and individuals. Check out our aid packages below
-              and click on the Donate button to get started.
-            </strong>
+            Together we can do more. We are accepting donations from both
+            organizations and individuals. Check out our aid packages below and
```

Comment:
```suggestion
            organizations and individuals. Check out <Link to="/medical-needs">medical needs</Link> and
```

Reply:
You have to import `Link` as follows:

import { Link } from "react-router-dom";